azureenergylabelerlib/datamodels.py

```diff
@@ -53,20 +53,40 @@
 class TenantEnergyLabelingData:  # pylint: disable=too-few-public-methods
     """Models the data for energy labeling to export."""
 
-    def __init__(self,
+    def __init__(self,  # pylint: disable= too-many-arguments
                  filename,
                  id,  # pylint: disable= redefined-builtin
-                 energy_label):
+                 energy_label,
+                 labeled_subscriptions,
+                 defender_for_cloud_findings):
         self.filename = filename
         self._id = id
         self._energy_label = energy_label
+        self._labeled_subscriptions = labeled_subscriptions
+        self._defender_for_cloud_findings = defender_for_cloud_findings
 
     @property
     def json(self):
         """Data to json."""
-        return json.dumps([{'Tenant ID': self._id,
-                            'Tenant Energy Label': self._energy_label}],
-                          indent=2, default=str)
+        subscription_metrics = []
+        for subscription in self._labeled_subscriptions:
+            energy_label = subscription.get_energy_label(self._defender_for_cloud_findings)
+            subscription_metrics.append({
+                'Subscription ID': subscription.subscription_id,
+                'Subscription Display Name': subscription.display_name,
+                'Number of high findings': energy_label.number_of_high_findings,
+                'Number of medium findings': energy_label.number_of_medium_findings,
+                'Number of low findings': energy_label.number_of_low_findings,
+                'Number of exempted findings': len(subscription.exempted_policies),
+                'Number of maximum days open': energy_label.max_days_open,
+                'Energy Label': energy_label.label
+            })
+        return json.dumps(
+            [{
+                'Tenant ID': self._id,
+                'Tenant Energy Label': self._energy_label,
+                'Labeled subscriptions': subscription_metrics
```

Comment:
Inconsistency with casing. subscriptions is lower case where all text is camel case.